tools/nvidia-gpu-lock.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+graphics=$(nvidia-smi --query-supported-clocks=graphics --format=csv | sed -n 2p | tr -d -c 0-9)
```

Comment:
imo we should have a windows version of this and also something for AMD if possible since this feature just won't work at all without this?